### ISSUE TITLE
Default translog durability is "request", but page about translog means "async"

### ISSUE THREAD

**Issue by m00t** [elastic/elasticsearch-definitive-guide#462]
Looks like <a href="https://github.com/elastic/elasticsearch-definitive-guide/blob/master/075_Inside_a_shard/50_Persistent_changes.asciidoc">this page</a> of guide means that default translog durability is async.
> Writes to a file will not survive a reboot until the file has been fsync'ed to disk. By default, the translog is fsync'ed every 5 seconds. Potentially, we could lose 5 seconds worth of data—if the translog were the only mechanism that we had for dealing with failure.

But <a href="https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules-translog.html">here</a> and <a href="https://github.com/elastic/elasticsearch/blob/master/core/src/main/java/org/elasticsearch/index/IndexSettings.java#L156">here</a> we see that it is "request".
```java
final String value = settings.get(INDEX_TRANSLOG_DURABILITY, Translog.Durability.REQUEST.name());
```

**Comment by polyfractal** [comment]
Yep, this changed in 2.0:  https://github.com/elastic/elasticsearch/pull/11011

Thanks for bringing it up, we'll make sure it is updated in the 2.x branch of the Guide!

**Comment by polyfractal** [comment]
Fixed in d79aa55a37f9795cd2b9f28b1cdda8b1ea069405.

Thanks!
